src/container/streamAnalyticsManager.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import StreamAnalyticsManagementClient = require("azure-arm-streamanalytics");
+import * as request from "request-promise";
+import * as vscode from "vscode";
+import { UserCancelledError } from "../common/UserCancelledError";
+import { StreamAnalyticsPicItem } from "../container/models/streamAnalyticsPickItem";
+import { AzureAccount, AzureSession } from "../typings/azure-account.api";
+
+export class StreamAnalyticsManager {
+    private readonly azureAccount: AzureAccount;
+    private readonly STREAM_ANALYTICS_IMAGE: string = "microsoft/azureiotedge-azure-stream-analytics:1.0.0-preview006";
```

Comment:
It seems 1.0.0-preview007 is the latest tag: https://hub.docker.com/r/microsoft/azureiotedge-azure-stream-analytics/tags/

Reply:
Do we need to specify the tag? Could we just use `latest` tag?

Reply:
I should remove this image URL, because we can retrieve it from publish rest API